dev-docs/source/UserSupport.rst

```diff
@@ -36,3 +36,66 @@ Bugs and Error Reports
 6.	When any issue is completed naming a user, that user is contacted to let them know it will be available in the nightly build and next release.  The gatekeeper that merges the fix should message the appropriate developer, to remind them to contact the original reporter. This could simply be through adding a comment while merging that points this out.
 
 
+Troubleshooting
+###############
+
+This is a list designed to take a user through how to gain diagnostic information, particularly when Mantid (Workbench) fails to launch. It may be appropriate to try certain steps first in certain cases.
+
+- Does the **splash screen** appear? Can get a rough idea how far through launch it stops.
+- Does the **error reporter** appear? Is there a useful stacktrace? [If the errorreport won't send, the user can check with "Show Details"]
+
+- Does **Qt** import correctly? In a command prompt, run the ``bin/mantidpython.bat`` script, and then ``import qtpy.QtCore``. Also, ``echo %PATH%`` could provide a clue.
+
+- Alternatively, try launching with: 
+
+.. code-block:: python
+
+    cd C:\MantidInstall\bin
+    set QT_PLUGIN_PATH=%CD%\..\plugins\qt5
+    python launch_workbench.pyw
+
+.. code-block:: python
+
+	cd C:\MantidInstall\bin
+	python -c "import sys; import os; import pprint; pprint.pprint(sys.path); pprint.pprint(os.environ)"
+
+- Try renaming **APPDATA**:
```

Comment:
Only `%APPDATA%` on Windows. On Linux this is in `$HOME/.config/mantidproject` and on MacOS is it is `~/Library/`